src/components/DroughtThresholds.vue

```diff
@@ -110,24 +125,39 @@ img {
 }
 #title-container {
   position: fixed;
+  background-image: url("./dry_drought_soil_publicDomain.jpg");
+  background-size: cover;
 }
 #scroll-container {
   z-index: 200;
 }
 .text-container {
   z-index: 500;
   border-radius: 25px;
-  background-color: #333534;
+  background: #C33F00;
+  //mix-blend-mode: overlay;
   max-width: 400px;
   p{
     padding: 25px;
   }
 }
+.knockout-text {
+  font-size: 20vh;
+  font-weight: bold;
+  font-family: Roboto, 'Helvetica Neue', Arial, sans-serif;
```

Comment:
Some of these fonts are not being used, right?

Reply:
I was always under the understanding that the font-family is always supposed to be a list so that the browser will try to match the first, but if it's not on the system, then it picks the second and so on until one works. Ending with sans-serif was standard practice, I thought. 

(I also didn't really think about the font specifically, I just used an example from the internet to start)

Reply:
That's more than I understood TBH, learn something new every day. Sounds good, and something to look into. In `App.vue` I directly import some fonts to use which is a way to make sure we have them.